source/_patterns/03-templates/04-header.mustache

```diff
@@ -98,4 +98,10 @@
 
 </div>
 
+{{^is_basket}}
 <div id="mobile-basket-button" class="basket-bar {{#basket.item_count}}hide-for-large-up{{/basket.item_count}}{{^basket.item_count}}hide{{/basket.item_count}}"><a href="{{basket_href}}" class="button">Basket (<span class="basket-item-count">{{basket.item_count}}</span>)</a></div>
+{{/is_basket}}
+
+{{#is_basket}}
```

Comment:
Is this value being set in every view that renders this? maybe we can hold off on this change?